### ISSUE TITLE
conda can't install latest version of pandas in GH workflows.

### ISSUE THREAD

**Issue by YarShev** [modin-project/modin#5256]
```
Encountered problems while solving:
    - nothing provides requested pandas 1.5.2
```

https://github.com/modin-project/modin/actions/runs/3533600573/jobs/5929433238

**Comment by YarShev** [comment]
https://github.com/conda-incubator/setup-miniconda/issues/264

**Comment by mvashishtha** [comment]
@YarShev can we close this?

**Comment by YarShev** [comment]
Yes, this is resolved in #5254.
